crate/pages/blog/release-v0-46-4.html

```diff
@@ -0,0 +1,119 @@
+title: New release: v0.46.4
+link: https://crate.io/blog/release-v0-46-4/
+author: Kenan
+description: Crate Release v0.46.4
+created: 2015/02/06 14:00
```

Comment:
Date format not correct, should be `yyyy-mm-dd`, `yyyy-mm-ddThh:mm` or `yyyy/mm/dd hh:mm:ss`